source/ed/ed_persistor.cpp

```diff
@@ -698,7 +718,7 @@ void EdPersistor::insert_stop_points(const std::vector<types::StopPoint*>& stop_
 }
 void EdPersistor::insert_lines(const std::vector<types::Line*>& lines){
     this->lotus.prepare_bulk_insert("navitia.line",
-            {"id", "uri", "name", "color", "code",
+            {"id", "uri", "name", "color", "code", "contributor_id",
```

Comment:
I think the frame at vehicle journey level is enough no ?

Why do we need contributor at line/route level ?